schema.sql

```diff
@@ -1,17 +1,20 @@
--- An author has many books
--- hello!
+-- Blog Database
 
-create table authors (
+-- Users Table
+CREATE TABLE users (
   id serial primary key,
-  name text
+  first_name text,
+  last_name text,
+  gender integer,
+  birthday timestamp
 );
 
-create table books (
-  id serial primary key,
+
+-- Posts Table
+CREATE TABLE posts (
   title text,
-  ISBN text,
-  cover_image text,
-  page_count integer,
-  author_id integer references authors (id)  
+  genre integer,
```

Comment:
Nice! The next thing you'd want to do is create a table that holds the "translations" of the genre numbers into genre names.